common/app/layout/Front.scala

```diff
@@ -476,8 +489,11 @@ object Front extends implicits.Collections {
         (Set.empty[TrailUrl], initialContext, emptyDedupedResultWithPath)
       ) { case ((seenTrails, context, dedupedFrontResult), (pressedCollection, index)) =>
         val omitMPU = pressedPage.metadata.omitMPUsFromContainers(edition)
+        val isNetworkFront = pressedPage.metadata.isNetworkFront
         val container = Container.fromPressedCollection(pressedCollection, omitMPU)
-        val (newSeen, newItems, (usedAndDeduped, usedButNotDeduped)) = deduplicate(seenTrails, container, pressedCollection.curatedPlusBackfillDeduplicated)
+        val isEditorialFront = ConfigAgent.getPriorityForPath(pressedPage.id).getOrElse("editorial") == "editorial"
```

Comment:
Sometimes the value returned for priority is an empty string. I checked in fronts tool and editorial was selected as the priority, so I presume empty string means editorial?